src/SFA.DAS.Roatp.Functions/ApplicationExtract.cs

```diff
@@ -224,10 +228,58 @@ public async Task<List<SubmittedApplicationAnswer>> ExtractAnswersForApplication
                     }
                 }
             }
+            //Extract QuestionId Y0-110 answer even if it is inactive, there is an issue setting answer active to false while adding orgnisation type answer to the tabular list as the last entry
+            var hasPartnershipAnswers = answers.Any(answer => answer.QuestionId == QuestionIdPartnership);
+            if(!hasPartnershipAnswers)
+            {
+                await ExtractPartnershipAnwers(applicationId, answers);
+            }
 
             return answers;
         }
 
+        private async Task ExtractPartnershipAnwers(Guid applicationId, List<SubmittedApplicationAnswer> answers)
+        {
+            try
+            {
+                var partnershipAnwers = await ExtractAnwersByQuestionTag(applicationId, AddPartners, QuestionIdPartnership);
```

Comment:
Don't see a point of `ExtractAnwersByQuestionTag`  method as there is null check repeated here which could be avoided.

Reply:
Code refactored